src/shared/locales/de/task.js

```diff
@@ -0,0 +1,78 @@
+export default {
+  'active': 'Aktive Downloads',
```

Comment:
This "downloading" is a bit too long, causing the layout to wrap. Is there a way to be shorter?